book/src/requirements.md

```diff
@@ -23,6 +23,11 @@ greater.
 Download and install the official pre-built binary from
 [LLVM download page](http://releases.llvm.org/download.html).
 
+You will also need to set `LIBCLANG_PATH` as an [environment
```

Comment:
Probably this is a `may`. If `llvm-config` is on your path then it shouldn't be needed... But I don't have a windows machine to check.

Also it's probably better to use `LLVM_CONFIG` instead of `LIBCLANG_PATH`, and let clang-sys infer the right thing. But again I'm not really familiar with what the best way to deal with llvm on Windows is, so this is ok for now.

Reply:
I am on windows, and that's just what the error messages told me to do when I tried `bindgen` the very first time and it didn't work XD